integreat_cms/cms/views/pages/page_tree_view.py

```diff
@@ -81,13 +85,33 @@ def get(self, request, *args, **kwargs):
                 "language_tree",
                 **{
                     "region_slug": region.slug,
-                }
+                },
             )
 
         if not request.user.has_perm("cms.change_page"):
-            messages.warning(
-                request, _("You don't have the permission to edit or create pages.")
-            )
+            access_granted_pages = Page.objects.filter(
+                Q(editors=request.user) | Q(publishers=request.user)
+            ).filter(region=request.region)
+            if request.user.organization:
+                access_granted_pages = access_granted_pages.union(
+                    Page.objects.filter(organization=request.user.organization)
+                )
+            if len(access_granted_pages) > 0:
+                page_list = ", ".join(
+                    translation.title
```

Comment:
I think it was cool showing the links to the edit forms, see e.g. an example here how it is done:
https://github.com/digitalfabrik/integreat-cms/blob/a5006d1c1f69a5d24a9cc4d8b424f9bc79844f2b/integreat_cms/cms/views/regions/region_actions.py#L47-L78